include/fmt/format-inl.h

```diff
@@ -86,8 +86,8 @@ inline int fmt_snprintf(char *buffer, size_t size, const char *format, ...) {
 # define FMT_SWPRINTF swprintf
 #endif // defined(_WIN32) && defined(__MINGW32__) && !defined(__NO_ISOCEXT)
 
-const char RESET_COLOR[] = "\x1b[0m";
-const wchar_t WRESET_COLOR[] = L"\x1b[0m"; 
+static FMT_CONSTEXPR_DECL const char RESET_COLOR[] = "\x1b[0m";
+static FMT_CONSTEXPR_DECL const wchar_t WRESET_COLOR[] = L"\x1b[0m"; 
```

Comment:
Why static?

Reply:
that would be copy-pasta mistake on my part.